core/templates/domain/utilities/language-util.service.ts

```diff
@@ -154,7 +156,7 @@ export class LanguageUtilService {
   }
   getAudioLanguageDescription(audioLanguageCode: string): string {
     const language = this.getSupportedAudioLanguages()[audioLanguageCode];
-    return language ? language.description : null;
+    return language ? language.description : '';
```

Comment:
I don't think that `''` is a correct replacement for `null`. The code should either be refactored so that both `null` and `''` are not needed (for example this function gets called only if we are 100% sure that the language always has `description` or `null` needs to be allowed in the type.

Reply:
`getAudioLanguageDescription` is being called in more than 4 files and the way I could think of to refactor this so that it would not return `null` is through declaring another function which checks if the `language` exists for the `audioLanguageCode` and then calling it each time before calling this function. 
```typescript
isSupportedAudioLanguageCode(audioLanguageCode: string): boolean {
   this.getSupportedAudioLanguages()[audioLanguageCode] ? true : false
}
```
But this required too big of a refactor and redundant code so I avoided it. But if you believe its worth the refactor then I can make a new PR for it.